### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by iah**
Changed a bit of words and translated some of the strings that still was in english.

### CODE REVIEW

**Comment by edgesince84** [comment]
Hi Isak, thanks for the PR.  I'll make sure someone gets eyes on this real soon!

File: res/values-sv/strings.xml [hunk 1]
```diff
@@ -11,56 +11,57 @@
...
-    <string name="PrefAutoUpdate">Automatisk Uppdatering</string>
+    <string name="PrefAutoUpdate">Automatisk uppdatering</string>
     <string name="PrefReadable">Läsbarhet</string>
     <string name="PrefGroupTor">Tor</string>
     <string name="PrefEnableTor">Aktivera Tor</string>
     <string name="PrefCheckOrbotStatus">Kontrollera Orbot-status</string>
     <string name="OrbotStatusOkTitle">Orbot-status</string>
     <string name="OrbotStatusOkMessage">Orbot verkar fungera bra!</string>
-    <string name="OrbotCheckSite">https://check.torproject.org</string>
+    <string name="OrbotCheckSite">https://check.torproject.org/string>
```
Comment (tagawa) [inline res/values-sv/strings.xml]: The closing `</string>` tag seems to be missing the first bracket.

File: res/values-sv/strings.xml [hunk 2]
```diff
@@ -11,56 +11,57 @@
     <string name="PrefGroupPrivacy">Sekretess</string>
     <string name="PrefGroupFeedback">Återkoppling</string>
     <string name="PrefTheme">Tema</string>
-    <string name="PrefThemeLight">Light</string>
-    <string name="PrefThemeDark">Dark</string>
+    <string name="PrefThemeLight">Ljus</string>
```
Comment (tagawa) [inline res/values-sv/strings.xml]: This should be `Ljust`

File: res/values-sv/strings.xml [hunk 3]
```diff
@@ -11,56 +11,57 @@
...
+    <string name="PrefThemeDark">Mörk</string>
```
Comment (tagawa) [inline res/values-sv/strings.xml]: This should be `Mörkt`

File: res/values-sv/strings.xml [hunk 4]
```diff
@@ -11,56 +11,57 @@
...
     <string name="OrbotStatusOkTitle">Orbot-status</string>
     <string name="OrbotStatusOkMessage">Orbot verkar fungera bra!</string>
-    <string name="OrbotCheckSite">https://check.torproject.org</string>
+    <string name="OrbotCheckSite">https://check.torproject.org/string>
     
-    <string name="ShareStory">Dela Berättelse</string>
-    <string name="ShareSearch">Dela Sök</string>
-    <string name="SharePage">Dela Sida</string>
+    <string name="ShareStory">Dela berättelse</string>
+    <string name="ShareSearch">Dela sök</string>
```
Comment (KarlHerler) [inline res/values-sv/strings.xml]: This should probably be "Dela sökning"

File: res/values-sv/strings.xml [hunk 5]
```diff
@@ -11,56 +11,57 @@
...
-    <string name="PrefClearCacheCookiesEveryLaunch">Varje Start</string>
+    <string name="PrefClearCacheCookiesEveryLaunch">Varje start</string>
     <string name="PrefClearCacheCookiesWeekly">Veckolig</string>
     <string name="PrefClearCacheCookiesMonthly">Månatlig</string>
-    <string name="PrefSendFeedback">Skicka Återkoppling</string>
-    <string name="PrefRate">Betygsätt App</string>
-    <string name="PrefAppSearch">Inkludera Apps</string>
+    <string name="PrefSendFeedback">Skicka feedback</string>
+    <string name="PrefRate">Betygsätt app</string>
+    <string name="PrefAppSearch">Inkludera apps</string>
```
Comment (matachi) [inline res/values-sv/strings.xml]: This should be `Inkludera appar`

**Comment by tagawa** [comment]
I've added a couple of review comments thanks to feedback from @ivh and https://twitter.com/AWNR

**Comment by tagawa** [comment]
Looks like the build is failing because it can't find a `.travis.yml` file. Maybe @fgei knows what to do?

**Comment by fgei** [comment]
I don't think we have Travis set up for the project, we usually do build with our own CI
is this PR ready to be tested and merged?

@nilnilnil are you interested in setting up Travis or are we fine the way we are now?
I thnk it could be useful at least for PR

**Comment by tagawa** [comment]
@fgei Oh, I see - that's OK then. We asked for review comments on Twitter about an hour ago so maybe give it a bit longer to see if we get more feedback.

**Comment by fgei** [comment]
ok, thanks @tagawa, no problem!

File: res/values-sv/strings.xml [hunk 6]
```diff
@@ -11,45 +11,46 @@
...
+    <string name="PrefClearCacheCookiesEveryLaunch">Varje start</string>
     <string name="PrefClearCacheCookiesWeekly">Veckolig</string>
     <string name="PrefClearCacheCookiesMonthly">Månatlig</string>
-    <string name="PrefSendFeedback">Skicka Återkoppling</string>
-    <string name="PrefRate">Betygsätt App</string>
-    <string name="PrefAppSearch">Inkludera Apps</string>
+    <string name="PrefSendFeedback">Skicka feedback</string>
+    <string name="PrefRate">Betygsätt app</string>
+    <string name="PrefAppSearch">Inkludera appar</string>
     <string name="PrefMainFontSize">Fontstorlek</string>
```
Comment (tagawa) [inline res/values-sv/strings.xml]: Please could this be changed to `Storlek på typsnittet` to be consistent with line 107?

Comment (iah) [inline res/values-sv/strings.xml]: Does it need to be? Since its not the same.

Comment (tagawa) [inline res/values-sv/strings.xml]: Yes, apparently so. We had a comment on Twitter: https://twitter.com/ericsson_jolle/status/735090054700339201

Comment (iah) [inline res/values-sv/strings.xml]: On it!

Comment (tagawa) [inline res/values-sv/strings.xml]: Thanks again

**Comment by fgei** [comment]
@thm: most of these string have just the first word capitalized (as Material guidelines I think), while on the English strings we have each word capitalized.
Should we change this to be consistent with the english strings, or just keep it this way?

**Comment by iah** [comment]
In my experience the "more" correct way in a grammar point of view in
Swedish, the first word is capitalised and not the second word.
 
*Best regards,*
*Isak Holmström* *isak@iah.io** **0705 / 32 38 20*
 
 
Den Tor 26 maj 2016 kl 12:35 skrev fgei:
> @thm[1]: most of these string have just the first word capitalized (as
> Material guidelines I think), while on the English strings we have
> each word capitalized.
>  Should we change this to be consistent with the english strings, or
>  just keep it this way?
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub[2]
>
 

Links:

  1. https://github.com/thm
  2. https://github.com/duckduckgo/android/pull/239#issuecomment-221836482

**Comment by ivh** [comment]
Yes, I liked that the patch de-capitalized all but the first words. It is much more uncommon in Swedish than in English to capitalize all.

**Comment by edgesince84** [comment]
@ivh thanks for the help, Thomas!  Native speakers' input is greatly appreciated 👍

**Comment by tagawa** [comment]
@fgei I think this is good to merge now if you're happy.

Thanks everyone for the feedback and corrections!

**Comment by fgei** [comment]
ok, thanks @tagawa, merging this now!
thanks everyone!

**Comment by edgesince84** [comment]
hey @iah thank you for your contribution to DuckDuckHack, we really appreciate it! I'm reaching out to let you know that we recently launched a new forum with the initiative of improving programming-related searches, and I hope you'll [join us](https://forum.duckduckhack.com/)!

There are a lot of important tasks up for grabs that we'd like your help with, so feel free to and [jump in](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53).

**Pull request merged**